src/utils/schema-formatter.ts

```
@@ -0,0 +1,178 @@
+function getZodTypeName(zodType: z.ZodTypeAny): string {
+  const typeName = (zodType as any)._def?.typeName;
+  
+  switch (typeName) {
+    case 'ZodString':
+      return 'string';
+    case 'ZodNumber':
+      return 'number';
+    case 'ZodBoolean':
+      return 'boolean';
+    case 'ZodArray':
+      const elementType = getZodTypeName((zodType as z.ZodArray<any>).element);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)